types/office-js-preview/index.d.ts

```diff
@@ -12566,7 +12564,7 @@ declare namespace Office {
          */
         id: string;
         /**
-         * Gets a value that indicates whether the attachment should be displayed in the body of the item.
+         * Gets a value that indicates whether an attachment appears as an image in the body of the item instead of in the attachment list.
```

Comment:
Since this object represents one attachment, I think we should be able to say "the attachment" (or "this attachment") in each property's description. Consider also updating attachmentType and the other properties accordingly. Here and in office-js version
 
```suggestion
         * Gets a value that indicates whether the attachment appears as an image in the body of the item instead of in the attachment list.
```